src/components/Footer.tsx

```diff
@@ -116,17 +116,10 @@ const Footer = () => {
 			<Container
 				style={{
 					color: 'white',
-					paddingBlockStart: '8rem',
+					// paddingBlockStart: '8rem',
```

Comment:
Like I'd said padding is better for surrounding space, while margin is better for distance from other items in the same tree level not parent. Nowadays using the `gap` attribute is preffered so that the parent has more control over the child's position within it.

Reply:
The padding from the container was much in my opinion. Without including the paddingBlockStart, there is still a default padding.